content/applications/productivity/iot/config/connect.rst

```diff
@@ -1,81 +1,143 @@
+      If no screen is attached to the :abbr:`IoT (Internet of Things)` box then the pairing code can
+      be accessed from the :abbr:`IoT (Internet of Things)` box admin console by clicking on
+      :guilabel:`POS Display`. For how to access the admin console visit
+      :ref:`iot_connect/token`
 
-#. Input the Pairing Code and click on the Pair button.
+#. Input the :guilabel:`Pairing Code` on the :abbr:`IoT (Internet of Things)` module of the database
+   and click on the :guilabel:`Pair` button.
 
-.. note::
-   Recent changes in modern web browsers forced us to modify the connection wizard.
-   If your screen is different from the screenshots, make sure that the
-   ``iot_pairing`` module is installed.
+.. important::
+   A secure HTTPS certificate will be created when the :guilabel:`pairing code` is confirmed. When
+   using a :guilabel:`token` to connect a HTTPS certificate will not be automatically created.
 
 WiFi Connection
 ===============
 
-#. Power on the IoT Box
+#. Connect all the devices that have to be connected with cables to the
+   :abbr:`IoT (Internet of Things)` box (ethernet, usb devices, etc.).
 
-#. Copy the token
+#. Power on the :abbr:`IoT (Internet of Things)` Box.
 
-   .. image:: connect/connect05.png
+#. Copy the :guilabel:`Token` from the :guilabel:`WiFi connection`section in
+   :menuselection:`IoT` module of the Odoo database.
 
-#. Connect to the IoT Box WiFi Network (make sure there is no ethernet
-   cable plugged in your computer).
+#. Connect to the :abbr:`IoT (Internet of Things)` Box WiFi Network (make sure there is no ethernet
+   cable plugged in the computer).
 
-   .. image:: connect/connect06.png
+   .. image:: connect/connect-iot-wifi.png
+      :align: center
+      :alt: WiFi networks available on the computer.
 
-#. You will be redirected to the IoT Box Homepage (if it doesn't work,
-   connect to the IP address of the box). Give a name to your IoT Box (not
-   required) and paste the token, then click on next.
+#. Upon connecting to the :abbr:`IoT (Internet of Things)` WiFi a browser will automatically
+   redirect to the :abbr:`IoT (Internet of Things)` Box Homepage. Give a name to the
+   :abbr:`IoT (Internet of Things)` Box, paste the previously copied token into the
+   :guilabel:`Server Token` field, and then click on :guilabel:`Next`.
 
-   .. image:: connect/connect07.png
+   .. image:: connect/server-token.png
+      :align: center
+      :alt: Enter the server token into the IoT box.
 
-   .. tip::
-      If you are on Runbot, do not forget to add the -all or -base in the
-      token (e.g. this token
-      **http://375228-saas-11-5-iot-f3f920.runbot16.odoo.com\|4957098401**
-      should become
-      **http://375228-saas-11-5-iot-f3f920-all.runbot16.odoo.com\|4957098401**).
+   .. note::
+      If the :abbr:`IoT (Internet of Things)` Box WiFi connection wizard doesn't start navigate to
+      the :abbr:`IoT (Internet of Things)` Box admin console page by entering the  
```

Comment:
missing sentence end ?

Reply:
Noted, I've created an anchor link here to the manual connection.